opm/simulators/wells/WellInterface_impl.hpp

```diff
@@ -1201,13 +1201,15 @@ namespace Opm
                               WellState& well_state,
                               Opm::DeferredLogger& deferred_logger)
     {
-        const int max_iter = param_.max_welleq_iter_;
+        const int max_iter = param_.max_inner_iter_wells_;
```

Comment:
Not sure whether we should change `max_iter` within this PR. It looks like they are a little different concepts here.

Reply:
If you want to have more consistency, probably you can use `assembleWellEq` instead of `assembleWellEqWithoutIteration`  for this function. Then `max_iter` should be the original one
```
const int max_iter = param_.max_welleq_iter_;
```

Reply:
The original plan was to remove max_welleq_iter completely. Thus the change. But lets keep it for now.